src/objects/list/list.twig

```diff
@@ -0,0 +1,14 @@
+{% set tag_name = tag_name|default('ul') %}
+
+{% if tag_name == 'ul' %}
+  {% set role = role|default('list') %}
```

Comment:
Not sure if it matters, but I expected to see this pipe surrounded by spaces, like this: 

`{% set role = role | default('list') %}`

Reply:
If I look at `button.twig`, It appears we _don't_ have a standard way for formatting this. 

https://github.com/cloudfour/cloudfour.com-patterns/blob/40f8e86d82037ff9b3cc78c6c5bab70b1d808526/src/components/button/button.twig#L1-L16

Is that correct?

Reply:
Not a release blocker for this PR!

Reply:
Either way works, but I've been writing it this way because it's how Twig's docs write it: https://twig.symfony.com/doc/2.x/templates.html